js/adapt-contrib-resources.js

```diff
@@ -46,9 +46,27 @@ class Resources extends Backbone.Controller {
         resources = resources.filter(resource => resource._isGlobal !== false);
       }
       model.set('_resources', resources);
+
+      this.setupTypes(model);
+
       drawer.triggerCustomView(new ResourcesView({ model }).$el);
     });
   }
+
+  setupTypes(model, resourcesData) {
+    const types = ['all']; // must contain 'all'
+
+    resourcesData._filterButtons.forEach((e, i) => {
```

Comment:
Is this a non-array object here? As it doesn't seem to work, even with the above